ttnn/cpp/ttnn/operations/data_movement/permute/permute.cpp

```diff
@@ -220,33 +206,14 @@ ttnn::Tensor ExecutePermute::invoke(
         return new_order;
     };
     auto itensor = (input_tensor.get_logical_shape().rank() < 4) ? ttnn::unsqueeze_to_4D(input_tensor) : input_tensor;
```

Comment:
unrelated to the PR. Can we make `unsqueeze_to_4D` call view inside to make it clear that this operation is basically free?